tests/visualization_tests/matplotlib_tests/test_parallel_coordinate.py

```diff
@@ -24,35 +26,95 @@ def test_plot_parallel_coordinate() -> None:
     # Test with no trial.
     study = create_study()
     figure = plot_parallel_coordinate(study)
-    assert len(figure.get_lines()) == 0
+    assert len(figure.get_figure().axes) == 0 + 1
     plt.savefig(BytesIO())
 
     study = prepare_study_with_trials(with_c_d=False)
 
     # Test with a trial.
     figure = plot_parallel_coordinate(study)
-    assert len(figure.get_lines()) == 0
+    axes = figure.get_figure().axes
+    assert len(axes) == 3 + 1
+    # axes[0] is the objective vertical line.
+    assert axes[0].get_ylim() == (0.0, 1.0)
+    # axes[1] is colorbar.
+    assert axes[1].get_ylabel() == "Objective Value"
+    assert axes[1].get_ylim() == (0.0, 1.0)
+    # axes[2] is `param_a`'s vertical line.
+    assert axes[2].get_ylim() == (1.0, 2.5)
+    # axes[3] is `param_b`'s vertical line.
+    assert axes[3].get_ylim() == (1.0, 2.0)
+    # Fetch line plots in parallel coordinate.
+    line_collections = figure.findobj(LineCollection)
+    assert len(line_collections) == 1
+    # Objective value as line plots.
+    objectives = [line[0, 1] for line in line_collections[0].get_segments()]
+    assert objectives == [0.0, 1.0]
+    expected_labels = ["Objective Value", "param_a", "param_b"]
+    xtick_labels = axes[0].get_xticklabels()
+    for expected_label, xtick_label in zip(expected_labels, xtick_labels):
+        assert expected_label == xtick_label.get_text()
```

Comment:
How about refactoring this to a helper function?

Reply:
Absolutely! af9aa71